### PR TITLE
update pageview to support adjust child size

### PR DESCRIPTION
**Pull request by liamcindy**
@zilongshanren @xiaofeng11 Please review this PR, thanks.

### CODE REVIEW

File: cocos/ui/UIPageView.h [hunk 1]
```diff
@@ -109,6 +109,12 @@ class CC_GUI_DLL PageView : public ListView
     static PageView* create();
 
     /**
+    * Create an empty PageView.
```
Comment (zilongshanren) [inline cocos/ui/UIPageView.h]: The comment should clarify the intent of `adjustChildSize` argument.

File: cocos/ui/UIPageView.cpp [hunk 2]
```diff
@@ -59,6 +59,19 @@ PageView* PageView::create()
     CC_SAFE_DELETE(widget);
     return nullptr;
 }
+
+PageView* PageView::create(bool adjustChildSize)
```
Comment (zilongshanren) [inline cocos/ui/UIPageView.cpp]: why pageview need the argument `adjustChildSize`?
We only need to call the `setAdjustChildSizeEnabled` in ListView?

**Comment by liamcindy** [comment]
@zilongshanren I've modify the problem that commented, please review it , thanks.

File: cocos/ui/UILayoutParameter.h [hunk 3]
```diff
@@ -283,12 +285,29 @@ class CC_GUI_DLL LinearLayoutParameter : public LayoutParameter
      */
     LinearGravity getGravity() const;
 
+    /**
+    * Sets size parameter for LayoutParameter.
+    *
+    * @see size
+    * @param size in LinearGravity.
+    */
+    void setLayoutItemSize(Size _size){ _normalSize = _size; }
```
Comment (zilongshanren) [inline cocos/ui/UILayoutParameter.h]: The argument shouldn't add _ prefix.  please change `_size` to `size`.
And the comments "@param size in LinearGravity." is also not clear, please fix it.

Comment (zilongshanren) [inline cocos/ui/UILayoutParameter.h]: I think it's better to also change `_normalSize` to `_layoutItemSize`.

File: cocos/ui/UILayoutParameter.h [hunk 4]
```diff
@@ -283,12 +285,29 @@ class CC_GUI_DLL LinearLayoutParameter : public LayoutParameter
      */
     LinearGravity getGravity() const;
 
+    /**
+    * Sets size parameter for LayoutParameter.
```
Comment (zilongshanren) [inline cocos/ui/UILayoutParameter.h]: `Sets size parameter for LayoutParameter.` is also not accurate.

**Comment by zilongshanren** [comment]
When fixing an issue,, don't introduce new unnecessary APIs.

**Comment by liamcindy** [comment]
@zilongshanren Please review the newly changes, thanks.

**Pull request merged**